lib/ansible/modules/cloud/amazon/rds_cluster_param_group.py

```diff
@@ -0,0 +1,298 @@
+module: rds_cluster_param_group
+short_description: Manages RDS Aurora Cluster Parameter Groups
+description:
+  -Manages the creation, modification, deletion of RDS Aurora parameter groups.
+version_added: "2.3"
+author: "Nick Aslanidis (@naslanidis)"
+options:
+  param_group_name:
+    description:
+      - The name of the DB cluster parameter group.
+    required: true
+    default: null
+  param_group_family:
+    description:
+      - The DB cluster parameter group family name.
+    required: false
+    default: null
+  description:
+    description:
+      - The description for the DB cluster parameter group.
+    required: false
+    default: null
+  params:
+    description:
+      - A list of parameters in the DB cluster parameter group
+    required: false
+    default: null
+  apply_method:
+    description:
+      - Indicates when to apply parameter updates.
+    required: false
+    default: null
+extends_documentation_fragment:
+  - aws
+  - ec2
+'''
+
+EXAMPLES = '''
+# Create RDS cluster param group with no parameters set
+- name: Create RDS cluster param group with no parameters set
+  rds_cluster_param_group:
+    region: ap-southeast-2
+    profile: auto_general_dev
+    state: absent
+    param_group_name: test-cluster-param-group
+    param_group_family: aurora5.6
+    description: test cluster param group
+  register: rds_cluster_param_group
+
+# Create RDS cluster param group with parameters
+- name: Create RDS cluster param group with no parameters set
+  rds_cluster_param_group:
+    region: ap-southeast-2
+    profile: auto_general_dev
+    state: absent
+    param_group_name: test-cluster-param-group
+    param_group_family: aurora5.6
+    description: test cluster param group
+    params: 
+      - ParameterName: binlog_format
+        ParameterValue: 'STATEMENT'
+        ApplyMethod: 'pending-reboot'
+
+      - ParameterName: binlog_checksum
+        ParameterValue: 'CRC32'
+        ApplyMethod: 'pending-reboot'
+  register: rds_cluster_param_group
+
+# Delete RDS cluster param group
+- name: Delete RDS cluster param group
+  rds_cluster_param_group:
+    region: ap-southeast-2
+    profile: auto_general_dev
+    state: absent
+    param_group_name: test-cluster-param-group
+  register: deleted_rds_cluster_param_group
+'''
+
+RETURN = '''
+DBClusterParameterGroups:
+  DBClusterParameterGroupName:
+    description: Provides the name of the DB cluster parameter group.
+    returned: when state=present
+    type: string
+    sample: "test-cluster-param-group"
+  DBParameterGroupFamily:
+    description: Provides the name of the DB parameter group family that this DB cluster parameter group is compatible with.
+    returned: when state=present
+    type: string
+    sample: "aurora5.6"
+  Description:
+    description: Provides the customer-specified description for this DB cluster parameter group.
+    returned: when state=present
+    type: string
+    sample: "test cluster param group"
+'''
+
+try:
+    import json
```

Comment:
No need  to wrap import json in a try block as we already assume python 2.6+ for boto